portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,31 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+/*
+* Scrolls to top of element with id "top"- top of page. Adjusts scroll position to account for
+* height for header/navigation bar
+*/
+function scrollToID(id) {
+  var y = document.getElementById(id).offsetTop - document.getElementById("header").offsetHeight;
+  //console.log("height of header " + document.getElementById("header").offsetHeight);
+  console.log("top of elm " + document.getElementById(id).offsetTop);
```

Comment:
Not a huge issue, but `console.log` should generally be removed after debugging.